sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/ManagementUtilities.cs

```diff
@@ -21,12 +21,14 @@ internal static class ManagementUtilities
         /// <param name="connectionScope"></param>
         /// <param name="managementLink"></param>
         /// <param name="amqpRequestMessage"></param>
+        /// <param name="transactionGroup"></param>
```

Comment:
nit:  You may want to consider filling these in or removing;  they're not providing any value in the current form.

Reply:
Yeah I will fix all the docs.